apps/studio/components/interfaces/Integrations/Vault/Secrets/AddNewSecretModal.tsx

```diff
@@ -23,10 +23,15 @@ const AddNewSecretModal = ({ visible, onClose }: AddNewSecretModalProps) => {
   const { mutateAsync: addKeyMutation } = usePgSodiumKeyCreateMutation()
   const { mutateAsync: addSecret } = useVaultSecretCreateMutation()
 
-  const { data: keys } = usePgSodiumKeysQuery({
-    projectRef: project?.ref,
-    connectionString: project?.connectionString,
-  })
+  const { data: keys } = usePgSodiumKeysQuery(
```

Comment:
nit: input field probably requires a loading state (ShimmeringLoader should do)

the dropdown shows "Create an encryption key" for a second, before the data is loaded